packages/shared/src/utils/currency/currency.js

```diff
@@ -16,8 +16,14 @@ export const formatMoney = (currency_value, amount, exclude_currency, decimals =
     let money = amount;
     if (money) money = String(money).replace(/,/g, '');
     const sign = money && Number(money) < 0 ? '-' : '';
-    const decimal_places = decimals || getDecimalPlaces(currency_value);
-
+    let decimal_places;
+    // Cryptos except IDK: minimum of 2, maximum of 8 decimal places without trailing zeros
+    if (isCryptocurrency(currency_value) && currency_value !== 'IDK') {
+        const without_trailing_zeros = money.replace(/(\d+\.\d*[^0])0+$/, '$1');
```

Comment:
```suggestion
        const without_trailing_zeros = money.replace(/(\d+\.\d*?)0+$/, '$1');
```

Reply:
If `1.00000000` should be changed to `1.00`, then we can use the suggested regex. Otherwise ignore the suggested regex

Reply:
Yes we need to.